### PR TITLE
Cleaned up the team members articles

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Team-Members/Technical-Contact.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ When you create a **New Project**, the user used to create the project will auto
 
 1. Go to the Project in the Umbraco Cloud Portal.
 2. Click **Settings** and go to **Edit Team**.
-3. In the **Technical Contact** section, click Add **Technical Contact**.
+3. In the **Technical Csontact** section, click Add **Technical Contact**.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Team-Members/Technical-Contact.md:20]:
```suggestion
3. Click Add **Technical Contact** in the **Technical Contact** section.
```

File: Umbraco-Cloud/Set-Up/Team-Members/Technical-Contact.md [hunk 2]
```diff
@@ -17,9 +17,11 @@ When you create a **New Project**, the user used to create the project will auto
...
+
     ![Add technical contact](images/add-technical-contact.png)
-4. In the **Add New Technical Contact** window, enter the **Name**, **Email** and **Telephone Number**.
+4. In the **Add New Technical Contact** window, enter the **Name**, **Email**, and **Telephone Number**.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Team-Members/Technical-Contact.md:23]:
```suggestion
4.  Enter the **Name**, **Email**, and **Telephone Number** in the **Add New Technical Contact** window.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Looks good to me💪 Just 2 suggestions to start the list item with an action word.

File: Umbraco-Cloud/Set-Up/Team-Members/index.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
 versionFrom: 9.0.0
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/Team-Members/index.md:2]:
```suggestion
versionFrom: 7.0.0
```

**Review by eshanrnh — APPROVED** [review]
Looks good 💪 Merging it in.

**Pull request merged**